app/helpers/display-message-time.js

```diff
@@ -10,7 +10,7 @@ export default Ember.Handlebars.makeBoundHelper(function(value) {
   if(!message_time) {
     return "";
   } else if(day_difference < 1) {
-    return "Today, "+moment(message_time).format('HH:mm');
+    return Ember.I18n.t("day.today")+", "+moment(message_time).format('HH:mm');
```

Comment:
@sunil-sharma can you use interpolation here? In chinese, it might be possible that the date comes before the word 'Today' so we need to provide that flexibility. E.g.

    return Ember.I18n.t("day.today", date=moment(message_time).format('HH:mm'))

and in i18n.js

    "today": "Today {{date}}"

We might need to do this in other places too.